### PR TITLE
Document how to customize the application domain in CAP (1.3.1 or higher, not 1.3).

### PR DESCRIPTION
**Pull request by andreas-kupries**
Ref: https://trello.com/c/Vr7di1lY/875-make-app-domains-configurable
Also: https://github.com/SUSE/scf/wiki/How-To-Customize-the-Application-Domains

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -314,6 +316,51 @@ Name            | Effect |
 `vagrant-box`    | Build the Vagrant box image using `packer` |
 `vagrant-prep`    | Shortcut for building everything needed for `make run` |
 
+# Deployment Customizations
+
+## Customize The Application Domain
+
+In a standard installation the domain used by applications pushed to
```
Comment (f0rmiga) [inline README.md]: Missing `,` after `installation`.

File: README.md [hunk 2]
```diff
@@ -314,6 +316,51 @@ Name            | Effect |
...
+CF is the same as the domain configured for CF itself.
```
Comment (f0rmiga) [inline README.md]: No need for `itself` at the end of the phrase.

File: README.md [hunk 3]
```diff
@@ -314,6 +316,51 @@ Name            | Effect |
...
+
+This document describes how to change this so that CF and applications
```
Comment (f0rmiga) [inline README.md]: Could be more specific to `this`. I would suggest:
`This document describes how to change this behaviour ...`

File: README.md [hunk 4]
```diff
@@ -314,6 +316,51 @@ Name            | Effect |
...
+use separate domains.
+
+:warning: The changes described below will work only for CAP 1.3.1,
+and higher.  They do not work for CAP 1.3, and will not work for the
```
Comment (f0rmiga) [inline README.md]: [nit] There is an extra space between the 2 sentences that shouldn't exist.

File: README.md [hunk 5]
```diff
@@ -314,6 +316,51 @@ Name            | Effect |
...
+CAP 2 series to be.
+
+1. Follow the basic steps for deploying UAA and SCF.
+1. When deploying SCF add a section like
```
Comment (f0rmiga) [inline README.md]: Missing `,` after `SCF`.

File: README.md [hunk 6]
```diff
@@ -314,6 +316,51 @@ Name            | Effect |
...
+
+   ```
+   bosh:
+     instance_groups:
+     - name: api-group
+       jobs:
+       - name: cloud_controller_ng
+         properties:
+           app_domains:
+           - APPDOMAIN
```
Comment (f0rmiga) [inline README.md]: Maybe `<APPDOMAIN>` so it is clearly a placeholder?

File: README.md [hunk 7]
```diff
@@ -314,6 +316,51 @@ Name            | Effect |
...
+
+After deployment use
+
+```
+cf curl /v2/info | grep endpoint
+```
+
+to verify that the CF domain is __not__ `APPDOMAIN`.
+
+Further verify, by pushing an application, that `APPDOMAIN` is printed
```
Comment (f0rmiga) [inline README.md]: Should be:
`Further, verify, by pushing an application, that ...` or `Further, by pushing an application, verify that ...`.

File: README.md (hunk 2 above)
Comment (andreas-kupries) [reply README.md]: ok

File: README.md (hunk 1 above)
Comment (andreas-kupries) [reply README.md]: maybe

File: README.md (hunk 3 above)
Comment (andreas-kupries) [reply README.md]: ok

File: README.md (hunk 4 above)
Comment (andreas-kupries) [reply README.md]: I have seen both one and two spaces in other docs, both some to be ok.
... ok, go one space. likely more consistent with rest of file.

File: README.md (hunk 5 above)
Comment (andreas-kupries) [reply README.md]: ok

File: README.md (hunk 6 above)
Comment (andreas-kupries) [reply README.md]: ok

File: README.md (hunk 7 above)
Comment (andreas-kupries) [reply README.md]: 1st form maybe, 2nd form looks ok to me

File: README.md (hunk 4 above)
Comment (f0rmiga) [reply README.md]: Yeah. If the entire file was using 2 spaces I could agree to just keep 2 spaces forever here. (or I would advocate for 1 space). :)

File: README.md (hunk 1 above)
Comment (f0rmiga) [reply README.md]: "An introductory word, phrase, or clause sets the stage for the sentence. Introductory clauses and long phrases must always be followed by a comma, while single words and short phrases may or may not. Use a comma when the phrase is a verbal phrase, a non-essential appositive, or an absolute phrase. Also, use a comma when a pause is necessary or the lack of a comma will cause confusion." - Tip when added this phrase to Grammarly.

File: README.md (hunk 7 above)
Comment (f0rmiga) [reply README.md]: 2nd form sounds more natural, indeed.

File: README.md (hunk 4 above)
Comment (jandubois) [reply README.md]: I think it makes perfect sense to have 2 spaces for text that is typically displayed in monospaced fonts. Once converted to HTML the double space is optimized away anyways.

So from a usability point of view I actually prefer 2 spaces after each sentence in Markdown documents and source code comments. 😛

File: README.md (hunk 7 above)
Comment (andreas-kupries) [reply README.md]: Done with commit 05aaa3c57e9da760304a8f1cd5e48362695b77d0

File: README.md (hunk 6 above)
Comment (andreas-kupries) [reply README.md]: Done with commit 05aaa3c57e9da760304a8f1cd5e48362695b77d0

File: README.md (hunk 5 above)
Comment (andreas-kupries) [reply README.md]: Done with commit 05aaa3c57e9da760304a8f1cd5e48362695b77d0

File: README.md (hunk 4 above)
Comment (andreas-kupries) [reply README.md]: Done with commit 05aaa3c57e9da760304a8f1cd5e48362695b77d0

File: README.md (hunk 3 above)
Comment (andreas-kupries) [reply README.md]: Done with commit 05aaa3c57e9da760304a8f1cd5e48362695b77d0

File: README.md (hunk 2 above)
Comment (andreas-kupries) [reply README.md]: Done with commit 05aaa3c57e9da760304a8f1cd5e48362695b77d0

File: README.md (hunk 1 above)
Comment (andreas-kupries) [reply README.md]: Done with commit fc7d98bce5778aa86bb05f50e32202f479227c24

**Pull request merged**